src/displayapp/screens/Dice.cpp

```diff
@@ -0,0 +1,231 @@
+#include "Dice.h"
+
+#include "Screen.h"
+#include "Symbols.h"
+#include "lvgl/lvgl.h"
+
+using namespace Pinetime::Applications::Screens;
+const uint8_t Dice::diceSizes[] = {2, 3, 4, 6, 8, 12, 20, 100};
+
+static void num_dice_inc_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->numDiceIncBtnEventHandler(event);
+}
+
+static void num_dice_dec_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->numDiceDecBtnEventHandler(event);
+}
+
+static void dice_size_inc_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->diceSizeIncBtnEventHandler(event);
+}
+
+static void dice_size_dec_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->diceSizeDecBtnEventHandler(event);
+}
+
+static void roll_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->rollBtnEventHandler(event);
+}
+
+void Dice::createScreen() {
+  resultCont = lv_cont_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(resultCont, 240, 80);
+  lv_obj_set_pos(resultCont, 0, 0);
+  lv_cont_set_layout(resultCont, LV_LAYOUT_COLUMN_LEFT);
+  lv_obj_set_style_local_pad_all(resultCont, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 10);
+
+  resultTotalLabel = lv_label_create(resultCont, nullptr);
+  lv_obj_set_size(resultTotalLabel, 220, 30);
+  lv_obj_set_style_local_text_color(resultTotalLabel, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+  lv_label_set_text(resultTotalLabel, "");
+
+  resultIndRollsLabel = lv_label_create(resultCont, nullptr);
+  lv_label_set_long_mode(resultIndRollsLabel, LV_LABEL_LONG_SROLL_CIRC);
+  lv_obj_set_size(resultIndRollsLabel, 220, 30);
+  lv_obj_set_style_local_text_color(resultIndRollsLabel, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+  lv_label_set_text(resultIndRollsLabel, "");
+
+  configCont = lv_cont_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(configCont, 240, 160);
+  lv_obj_set_pos(configCont, 0, 80);
+  lv_obj_set_style_local_bg_color(configCont, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+
```

Comment:
I don't think this is necessarily needed. Does it make the code cleaner?

Reply:
I suppose it's not necessary, but is it hurting anything?  It just helped me conceptually get everything organized during development.  I can rework it and remove it if that would be preferable.